controllers/operands/networkAddons_test.go

```diff
@@ -632,6 +639,37 @@ var _ = Describe("CNA Operand", func() {
 			}
 		}
 
+		kubevirtIpamTester := func(o ipamAnnotationParams) {
+			existingCNAO, err := NewNetworkAddons(hco)
+			Expect(err).ToNot(HaveOccurred())
+			if o.ipamExists {
+				existingCNAO.Spec.KubevirtIpamController = &networkaddonsshared.KubevirtIpamController{}
+			}
+
+			if o.setFeatureGate {
+				hco.Spec.FeatureGates.DeployKubevirtIpamController = ptr.To(o.featureGateValue)
+			}
+
+			cl := commontestutils.InitClient([]client.Object{hco, existingCNAO})
+			handler := (*genericOperand)(newCnaHandler(cl, commontestutils.GetScheme()))
+			res := handler.ensure(req)
+			Expect(res.UpgradeDone).To(BeFalse())
+			Expect(res.Err).ToNot(HaveOccurred())
+
+			foundCNAO := &networkaddonsv1.NetworkAddonsConfig{}
+			Expect(
+				cl.Get(context.TODO(),
```

Comment:
please use ginkgo's context; e.g.
```suggestion
		kubevirtIpamTester := func(ctx context.Context, o ipamAnnotationParams) {
			existingCNAO, err := NewNetworkAddons(hco)
			Expect(err).ToNot(HaveOccurred())
			if o.ipamExists {
				existingCNAO.Spec.KubevirtIpamController = &networkaddonsshared.KubevirtIpamController{}
			}

			if o.setFeatureGate {
				hco.Spec.FeatureGates.DeployKubevirtIpamController = ptr.To(o.featureGateValue)
			}

			cl := commontestutils.InitClient([]client.Object{hco, existingCNAO})
			handler := (*genericOperand)(newCnaHandler(cl, commontestutils.GetScheme()))
			res := handler.ensure(req)
			Expect(res.UpgradeDone).To(BeFalse())
			Expect(res.Err).ToNot(HaveOccurred())

			foundCNAO := &networkaddonsv1.NetworkAddonsConfig{}
			Expect(
				cl.Get(ctx,
```